src/scene/graphics/quad-render-utils.js

```diff
@@ -81,14 +81,14 @@ function drawQuadWithShader(device, target, shader, rect, scissorRect) {
 /**
  * Draws a texture in screen-space. Mostly used by post-effects.
  *
- * @param {import('../../platform/graphics/graphics-device.js').GraphicsDevice} device - The graphics device used to draw
+ * @param {import('../../platform/graphics/webgl/webgl-graphics-device.js').WebglGraphicsDevice} device - The graphics device used to draw
```

Comment:
AFAIK, `drawTexture` should/will work with WebGPU. So this should be a generic `GraphicsDevice`. @mvaligursky to confirm.

Reply:
yes this needs to stay as GraphicsDevice as it is multi-platform

Reply:
Theres a debug gaurd that stops that.  Can we define an abstract method for this and the "draw" method ?

Reply:
yes we should do that, similarly to how `setStencilState` does it. The problem is that JS does not have interfaces, and so if we change the function in Webgl class we have to remember to change it in the base class, there is not error reporting for this, which is bit limited. But we add those functions specifically for the documentation purpose, so we should do it here too.

Reply:
I don't see getCopyShader on WebGPU (called by drawTexture).  The debug warning might be correct, this looks like it will have issues on WebGPU.

Reply:
yep, at some point I'll clean this up, for now it does not work on WebGPU if you use the default shader (which is not a case when post-effects are used, as they supply a shader)

Reply:
I backed out this change.